src/tests/sqlsmith/tests/frontend/mod.rs

```diff
@@ -59,27 +78,34 @@ async fn create_tables(session: Arc<SessionImpl>, rng: &mut impl Rng) -> Vec<Tab
 
     for s in statements.into_iter() {
         let create_sql = s.to_string();
-        handle(session.clone(), s, create_sql).await;
+        handle(session.clone(), s, &setup_sql, &create_sql).await;
     }
 
     // Generate some mviews
     for i in 0..10 {
         let (sql, table) = mview_sql_gen(rng, tables.clone(), &format!("m{}", i));
+        setup_sql.push_str(&sql);
         let stmts = parse_sql(&sql);
         let stmt = stmts[0].clone();
-        handle(session.clone(), stmt, sql).await;
+        handle(session.clone(), stmt, &setup_sql, &sql).await;
         tables.push(table);
     }
-    tables
+    (tables, setup_sql)
 }
 
-fn test_batch_queries(session: Arc<SessionImpl>, tables: Vec<Table>, rng: &mut impl Rng) {
+#[allow(dead_code)]
+#[allow(unreachable_code)]
+#[allow(unused_variables)]
+#[allow(unused_mut)]
+fn test_batch_queries(
+    session: Arc<SessionImpl>,
+    tables: Vec<Table>,
+    rng: &mut impl Rng,
+    setup_sql: &str,
+) {
     for _ in 0..512 {
```

Comment:
After looking at output: https://buildkite.com/singularity-data/pull-request/builds/4715#0182449c-9a79-4e6a-98e7-a8a72d97a71f, perhaps it is not such a good idea to print to stderr.
Since we print all queries, when panic is encountered `nextest` will print the erroring query + setup code. However all previous queries and their setup code will also be printed to `stderr`, making it very messy.

I think best bet is still to `catch_unwind`. Note that this only deals with panics. But I think that's fine for now.

 If we realize it is possible crash the db in other ways, an even safer bet is to execute generated queries in a subthread, which executes the generated query. Even if it kills the process with `abort`, we can always recover from the parent thread.

cc @skyzh if you have other ideas

Reply:
Can we reduce number of queries running in a single test? 🤣 We can run hundreds of test cases of several queries instead of a few test casts of hundreds of queries. This can be easily done with libtest_mimic.